uint64 timestamp	# time since system start (microseconds)
uint64 error_count
uint32 pulse_width	# Pulse width, timer counts
uint32 period		# Period, timer counts
